tests/ErrorFormatter/SymplifyErrorFormatterVerboseTest.php

```diff
@@ -54,8 +54,10 @@ public static function getAdditionalConfigFiles(): array
 
     protected function getAnalysisResult($numFileErrors, int $numGenericErrors): AnalysisResult
     {
+        // @phpstan-ignore phpstanApi.constructor
```

Comment:
We don't use docblocks to avoid missed errors.
Could you put it to phpstan.neon?

Reply:
It's already removed in a later change